sdcm/utils/docker_remote.py

```diff
@@ -85,27 +85,6 @@ def receive_files(self, src, dst, **kwargs):  # pylint: disable=unused-argument
         result &= self.node.remoter.receive_files(remote_tempfile, dst, **kwargs)
         return result
 
-    def is_port_used(self, port: int, service_name: str) -> bool:
```

Comment:
How this code removal is related to the topic of the commit?

Reply:
It's a mistake, it should have been part of the work on docker backend support for running docker loaders, it's unneeded duplicate code (from days RemoteDocker didn't inherit BaseNode)

Reply:
I take it back, 

It is important to this change, since moving to newer version of scylla we no longer have `ss` installed as default,
the default implementation in BaseNode also install the if `ss` is missing.